Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs

```diff
@@ -43,6 +43,7 @@ public async Task<AuthenticationModel> GetTokenAsync(TokenRequestModel model)
                 authenticationModel.Email = user.Email;
                 authenticationModel.FirstName = user.FirstName;
                 authenticationModel.LastName = user.LastName;
+                authenticationModel.Avatar = "..\\Elevel.Api\\Resources\\default_avatar.jpg";
```

Comment:
bad practice to show path + in deployed version you will not have this folder. Please create wwwroot folder in root of WebApi project

Reply:
Done